code/modules/cargo/packs.dm

```diff
@@ -651,6 +651,8 @@
 					/obj/item/clothing/gloves/tackler/combat,
 					/obj/item/clothing/gloves/tackler/combat)
 	crate_name = "swat crate"
+	
+/datum/supply_pack/security/armory/woodstock // SKYRAT EDIT - ADDITION
```

Comment:
????????????

Reply:
lmao I think he forgot something there.

Reply:
it was overwritten in the aforementioned PR

Reply:
![image](https://user-images.githubusercontent.com/62520989/152919336-b74e8917-36c9-445f-92f2-24b97f239819.png)

Reply:
i just assumed it was done on accident